staging/src/k8s.io/apiserver/pkg/server/options/admission.go

```diff
@@ -94,3 +100,13 @@ func (a *AdmissionOptions) Validate() []error {
 	errs := []error{}
 	return errs
 }
+
+// SetDefaultPlugins sets the PluginNames field if it was not provided.
+// The field is populated from the hard coded list of default plugin names.
+// Note: This method MUST be called after AddFlags
+func (a *AdmissionOptions) SetDefaultPlugins() {
```

Comment:
This is good pull to talk against.  What if we made the `defaultPluginNames` into two fields on the `AdmissionOptions` that are set during initialization: `RecommendedPluginOrder` and `DefaultOffPlugins`.  If we did that, then we could use the values in `ApplyTo` if the cluster-admin hasn't set `--admission-control` manually.